includes/notifications/controllers/class.llms.notification.controller.failed.attempts.php

```diff
@@ -0,0 +1,133 @@
+<?php
+/**
+ * Notification Controller: Failed Attempts
+ *
+ * @package LifterLMS/Notifications/Controllers/Classes
+ *
+ * @since [version]
+ * @version [version]
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Notification Controller: Failed Attempts.
+ *
+ * @since [version]
+ */
+class LLMS_Notification_Controller_Failed_Attempts extends LLMS_Abstract_Notification_Controller {
+
+	/**
+	 * Trigger Identifier.
+	 *
+	 * @var string
+	 */
+	public $id = 'failed_attempts';
+
+	/**
+	 * Number of accepted arguments passed to the callback function.
+	 *
+	 * @since [version]
+	 * @var int
+	 */
+	protected $action_accepted_args = 2;
```

Comment:
```suggestion
	protected $action_accepted_args = 3;
```
since the `action_callback` below accepts and uses 3 params.